### DISCUSSION TITLE
Daily 13/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#607] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades
-

**Comment by rpgrca** [comment]
Ayer metí varios arreglos y terminé con casi todo lo de la venta, hoy hice el enbaldePago para usar en el checkout, varios refactoreos también. A la tarde tengo clase de web y vengo medio apretado en el laburo pero quiero ver si hago tiempo como para llegar a hacer lo del pago y terminar con esa parte.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve trabajando en la validación del mail y empecé con la tarea de vaciar el carrito.
Hoy tuve la mañana ocupada con estudios médicos. Continué con la tarea de vaciar el carrito aunque no la puedo terminar. Voy a subir la rama para revisión para ver que está mal.

**Comment by BraianTroncoso** [comment]
Ayer estuve finalizando todas las issues heredas de Contacto y me asigne un par más que finalicé. También tuve clases de Python. Hoy trabajé en la tarea de quitar boton y pude finalizarla, voy asignarme mas issues.
